### PR TITLE
[NFV] Add VSPerf integration tests

### PR DESCRIPTION
**Pull request by jlausuch**
VSPerf has some integration tests that don't need either traffic generator or physical NICs in the system. These do some basic operations on the vswitch. 

- Verification run: http://10.161.8.29/tests/63

### CODE REVIEW

**Comment by jlausuch** [comment]
@asmorodskyi  can you please review? Thanks.

File: tests/nfv/run_integration_tests.pm [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
```
Comment (asmorodskyi) [inline tests/nfv/run_integration_tests.pm]: it's already 2018 :)

File: tests/nfv/run_integration_tests.pm [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+sub run {
+    my $self        = shift;
+    my $vsperf_conf = "/etc/vsperf_ovs.conf";
+
+    select_console 'root-console';
+    mutex_create('nfv_trafficgen_ready');
+
+    # wait until traffic generator installation finishes
+    wait_for_children;
```
Comment (asmorodskyi) [inline tests/nfv/run_integration_tests.pm]: Currently you don't have multi-machine tests , and by this single line you will not introduce them. I am suggesting to add everything related to multi-machine tests in single commit. And for now just drop this line

File: tests/nfv/vsperf_installation.pm [hunk 3]
```diff
@@ -24,13 +24,15 @@ use utils;
 
 sub run {
     my $vsperf_repo = "https://gerrit.opnfv.org/gerrit/vswitchperf";
+    my $dpdk_repo   = "http://dpdk.org/git/dpdk";
```
Comment (asmorodskyi) [inline tests/nfv/vsperf_installation.pm]: with this change - this is not just vsperf installation , right ? how about renaming module to give more clear name ? Also summary of module needs to be updated.

File: tests/nfv/run_integration_tests.pm (hunk 2 above)
Comment (jlausuch) [reply tests/nfv/run_integration_tests.pm]: The tests are executed in 2 separate machines currently:
https://openqa.suse.de/tests/overview?distri=sle&version=15&build=410.1&groupid=133

File: tests/nfv/run_integration_tests.pm (hunk 1 above)
Comment (jlausuch) [reply tests/nfv/run_integration_tests.pm]: updated all to 2018

File: tests/nfv/vsperf_installation.pm (hunk 3 above)
Comment (jlausuch) [reply tests/nfv/vsperf_installation.pm]: I have moved it to the script that installs openvswitch and dpdk, it makes more sense there. Thanks for the suggestion.
Summary updated

**Comment by jlausuch** [comment]
Verification run with the latest changes http://10.161.8.29/tests/65

File: tests/nfv/openvswitch_dpdk.pm [hunk 4]
```diff
@@ -23,12 +24,14 @@ use utils;
...
 sub run {
+    my $dpdk_repo = "http://dpdk.org/git/dpdk";
+
     select_console 'root-console';
 
-    zypper_call('in openvswitch-switch', timeout => 200);
-    if (check_var('DPDK', 1)) {
-        zypper_call('in openvswitch-dpdk', timeout => 200);
-    }
+    zypper_call('in openvswitch-switch dpdk qemu git-core', timeout => 200);
```
Comment (asmorodskyi) [inline tests/nfv/openvswitch_dpdk.pm]: currently openvswitch_dpdk and vsperf_installation doing a lot of thing in common : 
1. both install some packages needed by "run_integration_tests" ( they even try to install same package - git-core ) 
2. both clone repositories needed by "run_integration_tests"
3. they run one after another 

keep all this in mind my question is - is there chance that they will run independently ? Or probably it make sense to combine them in single module with name "before_integration_tests" ?

File: tests/nfv/run_integration_tests.pm (hunk 2 above)
Comment (asmorodskyi) [reply tests/nfv/run_integration_tests.pm]: acknowledged, let's leave it like this

File: tests/nfv/openvswitch_dpdk.pm (hunk 4 above)
Comment (asmorodskyi) [reply tests/nfv/openvswitch_dpdk.pm]: agreed with @jlausuch to do this in separate PR

**Pull request merged**